docs/docs_style_guide.md

```diff
@@ -0,0 +1,479 @@
+  don't separate with a comma etc)
+
+* 10 November to 21 December
+
+Don't use 'quarter' for dates; use the months, for example: 'Jan to
+Mar 2013'.
+
+## E
+
+### eg, etc and ie
+
+Don't use full stops after or between these notatinos.
+
+You can use the long form ('for example' instead of 'eg',
+'specifically' instead of 'ie' etc) if you like. Some people are not
+familiar with abbreviations such as eg, so consider your audience
+before abbreviating.
+
+### email
+
+One word.
+
+### Email addresses
+
+Write email addresses in full, in lower case and as active links.
+Don't include any other wards as part of the link. Don't label email
+addresses ie don't write 'Email: name@example.org'.
+
+### etc
+
+See 'eg, etc and ie'.
+
+## F
+
+### FAQs (frequently asked questions)
+
+Don't use FAQs in LiveCode documentation. If you write content by
+starting with user needs, you won't need to use FAQs.
+
+## H
+
+### homepage
+
+Lower case.
+
+### Hyphenation
+
+Hyphenate:
+
+* 're-' words starting with 'e', eg 're-evaluate'
+* co-ordinate
+* co-operate
+
+Don't hyphenate:
+
+* reuse
+* reinvent
+* reorder
+* reopen
+* email
+
+Don't use a hyphen unless it's confusing without it. For example, a
+little used-car is different from a little-used car.
+
+Use 'to' for time and date ranges, not hyphens.
+
+## I
+
+### ie
+
+See 'eg, etc and ie'
+
+### internet
+
+Lower case
+
+### Italics
+
+Don't use italics. Use 'single quotation marks' if referring to a
+document, book, or standard.
+
+## L
+
+### Links
+
+Front-load your link text with the relevant terms and make them active
+and specific.
+
+### Lists
+
+Lists should be bulleted to make them easier to read. See 'bullets
+and steps'.
+
+Very long lists can be written as a paragraph with a lead-in sentence
+if it looks better. Eg 'The following countries are in the EU: Spain,
+France, Italy...'
+
+## M
+
+### Mathematical content
+
+Use a minus sign for negative numbers: '−6'
+
+Ratios have no space either side of the colon: '5:12'
+
+One space each side of symbols: +, −, ×, ÷ and =, eg '2 + 2 = 4'
+
+Use the minus sign for subtraction. Use the correct symbol for the
+multiplication sign (−×) not the letter x.
+
+Write out and hyphenate fractions: two-thirds, three-quarters
+
+Write out decimal fractions as numerals. Use the same number format
+for a sequence: 0.75 and 0.45
+
+### Measurements
+
+Use SI units (not Imperial units).
+
+Use numerals and spell out measurements at first mention.
+
+Don't use a space between the numeral and abbreviated measurement:
+'3,500kg' not '3,500 kg'.
+
+Abbreviating kilograms to kg is fine.
+
+If the measurement is more than one word, eg 'kilometres per hour',
+then spell it out the first time it is used with the abbreviation.
+From then on, abbreviate. If it is only mentioned once, don't
+abbreviate.
+
+Use Celcius for temperature, eg 37℃.
+
+### Metaphors
+
+See 'words to avoid'.
+
+### Millions
+
+Always use million in money (and billion), eg '£138 million'. Use
```

Comment:
"billion" is a bit of a problem, as there are two distinct definitions of the term which differ by a factor of a thousand. I'd avoid the word altogether unless it's unavoidable, and in that case make it explicit which version is being used. The terms "thousand million" and "million million" are more explicit but awkward.